frontend/server/src/DAO/Contests.php

```diff
@@ -157,7 +161,9 @@ final public static function getByAliasWithExtraInformation(string $alias): ?arr
     final public static function getByProblemset(
         int $problemsetId
     ): ?\OmegaUp\DAO\VO\Contests {
-        $sql = 'SELECT * FROM Contests WHERE problemset_id = ? LIMIT 0, 1;';
+        $sql = 'SELECT ' .
+        join(', ', array_keys(\OmegaUp\DAO\VO\Contests::FIELD_NAMES)) . ' ' .
+        'FROM Contests ' . 'WHERE problemset_id = ? LIMIT 0, 1;';
```

Comment:
```suggestion
        join(', ', array_keys(\OmegaUp\DAO\VO\Contests::FIELD_NAMES)) . 
        ' FROM Contests ' . 'WHERE problemset_id = ? LIMIT 0, 1;';
```

acá nos podemos ahorrar una concatenación

Reply:
También en esta consulta podrías agregar la variable extra, para poder concatenar de manera más clara. Por ejemplo:
```php
        $fields = join(', ', array_keys(\OmegaUp\DAO\VO\Contests::FIELD_NAMES));
        $sql = 'SELECT 
                    ' . $fields . '
                FROM
                    Contests
                WHERE
                    problemset_id = ?
                LIMIT 0, 1;';
```